binding/lua/docs/BENCHMARK.md

```diff
@@ -0,0 +1,44 @@
+#Multiverso Torch Binding Benchmark
+
+## Task Description
+
+Perform CIFAR-10 classification with torch resnet implementation.
+
+## Codebase
+
+[Microsoft/fb.resnet.torch multiverso branch](https://github.com/Microsoft/fb.resnet.torch/tree/multiverso)
+
+## Hardware
+
+- **Hosts** : 1
+- **GPU** : GeForce GTX TITAN X * 4
+- **CPU** : Intel(R) Core(TM) i7-5960X CPU @ 3.00GHz
+- **Memory** : 128GB
+
+## Common settings
+
+- batchSize 128
+- depth 32
+- nEpochs 164
+
+## Clarification for multiverso settings
+
+- Epoch means epoch for each worker with multiverso.
```

Comment:
I think both are OK in case we make it clear. I just don't want to mislead the results shown in the following pictures.